client/blindai/client.py

```diff
@@ -547,6 +547,7 @@ def upload_model(
         dtype_out: ModelDatumType = ModelDatumType.F32,
         sign: bool = False,
         model_name: Optional[str] = None,
+        sealed: bool = False,
```

Comment:
i think a argument name like `save_model` is better here, since `seal` might not be that clear for data-scientists.
Also, change the python doc for sealed, so that any user can understand that it means that the model is saved to the disk in the server side, and will be kept across server restarts ; and that the model is "sealed", => stored encrypted so that only the enclave can unseal it.